### DISCUSSION TITLE
Possible bug? head table version number seemingly being misread

### DISCUSSION THREAD

**Discussion by arrowtype** [fonttools/fontbakery#4213] [Q&A]
This may be related to https://github.com/googlefonts/fontbakery/issues/2403, but I’m not sure.

When I run check `com.google.fonts/check/font_version`, I receive the following `FAIL` message:

```
 >> com.google.fonts/check/font_version
    Checking font version fields (head and name table).
    with fonts__trial/ArrowType-NameSans-v011-Trial/AT-NameSans-Trial/AT-NameSansVariable.Trial.ttf


     FAIL head version is "1.17900" while name version string                   
          (for platform 3, encoding 1) is "Version 0.11, Build 1.179            
          [eee8b8f564]". [code: mismatch]                                       


    Result: FAIL
```

However, if I inspect those two tables using TTX, they appear to match exactly:

```
  <head>
    <!-- skipping other fields -->
    <fontRevision value="1.179"/>
    <!-- skipping other fields -->
  </head>
```

```
  <name>
    <!-- skipping other fields -->
    <namerecord nameID="5" platformID="3" platEncID="1" langID="0x409">
      Version 0.11, Build 1.179 [eee8b8f564]
    </namerecord>
    <!-- skipping other fields -->
  </name>
```

I am using FontBakery version `0.8.13`.

What might be causing this? 

Thanks for any info!

---

Btw, I can share the slightly limited "trial" font as a binary to test against, if it’s useful:

[AT-NameSans-Trial.zip](https://github.com/googlefonts/fontbakery/files/12149188/AT-NameSans-Trial.zip)

As a note, this is the trial version of [a retail font](https://www.futurefonts.xyz/arrowtype/name-sans), so folks are free to use it for testing and evaluation (include for evaluating a FontBakery check like this one), but it can’t be used for published logos and graphic design, etc ... please see the included license for details. The trial font excludes some characters and is named to try to keep this clear to potential users.

For what it’s worth, the non-trial version exhibits similarly unexpected behavior in FontBakery.

Thanks for any eyes on this!

**Comment by felipesanches** [comment]
`"Version 0.11, Build 1.179 [eee8b8f564]"` is not `version="1.179"`, even though I can see that you're using the build number as numerically equivalent to version number. That's the aspect of it that is confusing the check.

**Reply by arrowtype** [reply]
Ohh, I see. Thanks for helping me understand what the check was really saying.

It does still seem non-ideal that it misreports the number in the HEAD table (`FAIL head version is "1.17900"`), as that confused me about where the problem was. 

Maybe it might be clearer as something like: 

`Head version is "1.179" so "Version 1.179" is expected in the name version string (for platform 3, encoding 1)`

Anyway, thanks again for helping me to better understand this!

**Reply by felipesanches** [reply]
yeah, I think I can improve it that way
